java/java.editor.base/src/org/netbeans/modules/java/editor/base/semantic/ColoringAttributes.java

```diff
@@ -30,7 +30,9 @@
 public enum ColoringAttributes {
 
     UNUSED,
-
+    SEALED,
```

Comment:
These new constants don't seem to be needed, right? The keywords should have attribute KEYWORD, I think, and that should be enough to make them highlighted as a keyword.